dspot/src/main/java/eu/stamp_project/dspot/mongodb/DspotInformationCollector.java

```diff
@@ -0,0 +1,10 @@
+package eu.stamp_project.mongodb;
+
+import com.martiansoftware.jsap.JSAPResult;
+
+public interface DspotInformationCollector {
```

Comment:
interface are essential to be documented. add one sentence of doc for the class and per method, ideally with an example